.python-version

```diff
@@ -0,0 +1 @@
+3.8.12
```

Comment:
This makes it easier since we have `pre-commit` install under that Python installation.